frontend/src/components/Place/DeletePlaceButton.tsx

```
@@ -0,0 +1,17 @@
+
+type DeletePlaceButtonProps = {
+  placeId: number;
+};
+
+const DeletePlaceButton = ({ placeId }: DeletePlaceButtonProps) => {
+  const id = Number(useParams<string>().id);
+  const deletePlaceMutation = useDeletePlaceMutation();
+  const onClick = () => {
+    deletePlaceMutation.mutate({ id, placeId });
+  };
+  return <button onClick={onClick}>삭제</button>;
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)